docs/tutorials/05_torch_connector.ipynb

```diff
@@ -833,9 +805,9 @@
     "data_iter = iter(train_loader)\n",
     "fig, axes = plt.subplots(nrows=1, ncols=n_samples_show, figsize=(10, 3))\n",
     "\n",
-    "while n_samples_show > 0:\n",
-    "    images, targets = data_iter.__next__()\n",
-    "\n",
+    "for images, targets in data_iter:\n",
+    "    if n_samples_show <= 0:\n",
+    "        break\n",
```

Comment:
Would it be possible to change the `if` statement to `if count == n_samples_show:` just to consistency with another cell?